src/plugins/support/InvocationTraceSupport.ts

```diff
@@ -39,9 +53,12 @@ class InvocationTraceSupport {
         }
     }
 
-    static generateResourceIdFromSpan(span: ThundraSpan): string {
+    static generateResourceIdFromSpan(span: ThundraSpan, entryId?: string): string {
         if (span.className && span.operationName) {
             let id = `${span.className.toUpperCase()}\$${span.operationName}`;
+            if (entryId) {
```

Comment:
If `resourceName` (new name of `entryId` parameter) is passed, `operationName` should not be used in the generated entry id.
```
if (!resourceName) {
    resourceName = span.operationName;
}
let id = `${span.className.toUpperCase()}\$${resourceName}`;